integration-tests/src/test/java/oracle/kubernetes/operator/ItMonitoringExporter.java

```diff
@@ -260,12 +254,19 @@ private static void executeShelScript(String srcLoc, String destLoc, String file
             + args
             + " | tee "
             + outLogFile;
-    TestUtils.execOrAbortProcess(crdCmd, true);
-    crdCmd = " cat " + destLoc + "/" + outLogFile;
-    ExecResult result = ExecCommand.exec(crdCmd);
-    assertFalse(
-        result.stdout().contains("BUILD FAILURE"), "Shell script failed: " + result.stdout());
+    ExecResult result = TestUtils.execOrAbortProcess(crdCmd, true);
+    if (result.exitValue() != 0) {
```

Comment:
if the exit value was not zero, you will never get to this line.
if you need the result, you have to use `ExecCommand.exec()` instead

Reply:
replaced with execCommand.exec()